CONTRIBUTING.md

```diff
@@ -88,7 +88,26 @@ docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:nightly-c
 
 bazel test -c opt -k \
 --test_timeout 300,450,1200,3600 \
---test_output=errors \
+--test_output=all \
+//tensorflow_addons/...
+```
+
+### Locally Testing GPU
```

Comment:
Maybe using #### which is same with above?

Reply:
Good catch thanks!